extensions/arc/src/ui/components/keyValueContainer.ts

```diff
@@ -116,7 +116,7 @@ export abstract class BaseInputKeyValue extends KeyValue {
 			height: '17px'
 		}).component();
 
-		this.disposables.push(copy.onDidClick(async () => {
+		this.disposables.push(copy.onDidClick!(async () => {
```

Comment:
This kinda sucks that this has to be done but every event getter in our exthost is written as 
```
get event (): vscode.Event {
    const emitter = map.get(id);
    return emitter && emitter.event;
}
```
which means we are accounting for the fact that the emitter may not exist and therefore every event may not exist. (`emitter && emitter.event` will return undefined if emitter is undefined). I'm not familiar enough with the code to say what the right solution here is @Charles-Gagnon @alanrenmsft